OpenRA.Mods.Common/Traits/Render/WithBuildingRepairDecoration.cs

```diff
@@ -47,10 +47,7 @@ protected override bool ShouldRender(Actor self)
 
 		protected override PaletteReference GetPalette(Actor self, WorldRenderer wr)
 		{
-			if (!info.IsPlayerPalette)
-				return wr.Palette(info.Palette);
-
-			return wr.Palette(info.Palette + rb.Repairers[shownPlayer % rb.Repairers.Count].InternalName);
+			return wr.Palette(info.IsPlayerPalette ? info.Palette + rb.Repairers[shownPlayer % rb.Repairers.Count].InternalName : info.Palette);
```

Comment:
These changes seem to be no longer necessary?

Reply:
How so?

Reply:
Yes, the logic is the same. But the overall goal of the PR is to simplify usage of `WorldRenderer.Palette` method, so the clients of this method don't have to deal with null or empty string as the palette name.